### PR TITLE
EW properties API and deletion API support

### PR DESCRIPTION
**Pull request by stingare**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -365,6 +366,20 @@ Usage: `akamai list-contracts`
...
+Usage: `akamai edgeworkers list-properties <edgeworkerId> [options]`
+
+| Option | Description |
+| - | - |
+| -h, --help  | output usage information |
+| --activeOnly | Returns only active properties |
+
+| Argument | Existence | Description |
+| - | - | - |
+| edgeworkerId | required | Edgeworker identifier.
```
Comment (swathimr) [inline README.md:381]: "edgeworkerId" and the one listed above "lp `<edgeworker-identifier> [options]`" are different. can we keep it same ?

File: package-lock.json [hunk 2]
```diff
@@ -1474,9 +1474,7 @@
       "version": "3.1.2"
     },
     "lodash": {
-      "version": "4.17.21",
-      "resolved": "https://registry.npmjs.org/lodash/-/lodash-4.17.21.tgz",
-      "integrity": "sha512-v2kDEe57lecTulaDIuNTPy3Ry4gLGJ6Z1O3vE1krgXZNrsQ+LFTGHVxVjcXPs17LhbZVGedAJv8XZ1tvj5FvSg=="
+      "version": "4.17.20"
```
Comment (swathimr) [inline package-lock.json:1477]: was this downgraded for a reason ?

File: src/edgeworkers/ew-handler.ts [hunk 3]
```diff
@@ -200,6 +200,27 @@ export async function getContracts() {
   }
 }
 
+export async function getProperties(ewId: string, activeOnly: boolean) {
+  let properties = await cliUtils.spinner(edgeWorkersSvc.getProperties(ewId, activeOnly), `Retrieving properties for EdgeWorker Id ${ewId}...`);
+  if (properties && !properties.isError) {
+    if (properties.properties.length > 0) {
```
Comment (swathimr) [inline src/edgeworkers/ew-handler.ts:206]: nitpick. "properties.properties.length" sounds odd. can we say propList.properties.length. Either propList or whatever the api is returning

File: src/edgeworkers/ew-service.ts [hunk 4]
```diff
@@ -103,6 +103,14 @@ export function getContracts() {
   return httpEdge.getJson(`${EDGEWORKERS_API_BASE}/contracts`).then(r => r.body).catch(err => error.handleError(err,"GET_CONTRACT"));
 }
 
+export function getProperties(ewId: string, activeOnly: boolean) {
+  let qs: string = "?activeOnly=true";
+  if (activeOnly === undefined) {
+    qs = "";
+  }
```
Comment (swathimr) [inline src/edgeworkers/ew-service.ts:110]: can we reverse this ? For instance lets define it with empty string and if its not undefined then add this `"?activeOnly=true";`

File: package-lock.json (hunk 2 above)
Comment (stingare) [reply package-lock.json:1477]: package-lock.json was automatically updated upon running npm install I believe this has to do with the node_modules/lodash version being listed as 4.17.20

**Review by swathimr — APPROVED** [review]
LGTM

**Pull request merged**